app/models/reservation.rb

```diff
@@ -361,6 +361,18 @@ def update_billable_minutes
     update_column(:billable_minutes, calculated_billable_minutes)
   end
 
+  def billable_duration_mins
```

Comment:
@jossim I realized you had made the wrong method public here - it's `billable_duration_mins` we need to reference in statement creation